admin/spam/urls.py

```diff
@@ -4,6 +4,9 @@
 
 urlpatterns = [
     url(r'^$', views.SpamList.as_view(), name='spam'),
-    url(r'^(?P<spam_id>[a-z0-9]+)/$', views.SpamDetail.as_view(), name='detail'),
-    url(r'^(?P<spam_id>[a-z0-9]+)/email/$', views.email, name='email'),
+    url(r'^user/id-(?P<user_id>[a-z0-9]+)/$', views.UserSpamList.as_view(),
```

Comment:
Yes. As addressed in two or three other comments on PRs right now this will be fixed in a future PR where this can be taken care of all at once. @sloria @GaryKriebel Basically this was a bad decision that has propagated to a dozen PRs. To fix one way in one and a slightly different way in another will cause more merge conflicts then I am already handling (19+ files). This will be a simple fix when more PRs are merged and will not effect usage in the few days that it might actually get to be used.